src/pages/Auth/helper.tsx

```diff
@@ -12,18 +12,18 @@ export const authForm: FormProps = {
 			type: INPUT_TYPE.EMAIL,
 			placeHolder: 'User name',
 			required: true,
-            id: '01'
+			id: '01',
 		},
 		{
 			name: 'password',
 			type: INPUT_TYPE.PASSWORD,
 			placeHolder: 'Password',
 			required: true,
-            id: '02'
+			id: '02',
 		},
 	],
-    button: {
-        text: 'Submit',
-        clickHandler: () => {},
-    },
+	button: {
+		text: 'Submit',
+		clickHandler: () => {},
```

Comment:
doesn't look good if you need to pass an empty function just to make form work eventhough the action managed elsewhere.

Reply:
Since this prop is optional, I simply removed it.